WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AuthenticatedWebView.swift

```diff
@@ -1,6 +1,7 @@
 import SwiftUI
 import WebKit
 import Alamofire
+import Networking
```

Comment:
Nit: maybe we should inject just the UserAgent class only?
```suggestion
import class Networking.UserAgent
```